config/boards/qemu-uefi-x86.conf

```diff
@@ -0,0 +1,24 @@
+# x86_64 via UEFI/BIOS for generic virtual board
+#
+# Usage: Use this board to run armbian on a
+#        virtualized environment (eg: QEMU/KVM)
+#
+# Notes:
+# - Differences with the 'uefi-x86' board:
+#   - support kernel boot messages on graphical
+#     and console/serial devices
+#   - support prompt on graphical/console devices
+#   - Patches targeting virtualized env on x86
```

Comment:
"Patches", meaning kernel patches, are per-kernel, and not per-board, so this comment is a bit misleading.